SampleApps/CurrencyRecogniser/CurrencyRecogniser.iOS/AppDelegate.cs

```diff
@@ -14,6 +15,9 @@ namespace CurrencyRecogniser.iOS
     [Register("AppDelegate")]
     public partial class AppDelegate : global::Xamarin.Forms.Platform.iOS.FormsApplicationDelegate
     {
+        bool useFilesForTheModel = false;
```

Comment:
@jimbobbennett 
I added this flag to switch between assets and files for the model